src/main/resources/templates/db_scripts/schema.json

```diff
@@ -185,6 +191,11 @@
       "tableName": "settings",
       "fromModuleVersion": "mod-organizations-storage-4.7.0",
       "withMetadata": true
+    },
+    {
+      "tableName": "privileged_contacts",
+      "fromModuleVersion": "mod-organizations-storage-4.7.0",
+      "withMetadata": true
```

Comment:
We have some tickets in backlog to enable optimistic locking for existing tables like this: https://issues.folio.org/browse/MODORGSTOR-106
I think we also should enable it for new tables.
I think we need to do the same for organization_types table

Reply:
Lets check in architects do we need to enable optimistic locking for new tables

Reply:
enabled for new tables